code/__defines/dna.dm

```diff
@@ -80,3 +80,21 @@ var/HALLUCINATIONBLOCK = 0
 var/NOPRINTSBLOCK      = 0
 var/SHOCKIMMUNITYBLOCK = 0
 var/SMALLSIZEBLOCK     = 0
+
+
+// Hair Lengths
+
+var/list/hairnone = list("Short Hair", "Short Hair 2", "Cut Hair", "Skinhead", "Balding Hair", "The Family Man", "Flaired Hair", "Crewcut", "Combover", "Overeye Short", "Overeye Very Short, Alternate", "Hime Cut", "Hime Cut Alt", "Short Hime Cut", "CIA", "Mulder", "Scully", "Wheeler", "Bangs Short", "Short Bangs", "Low Fade", "Balding Fade", "No Fade", "Trimmed Flat Top", "Shaved", "Trimmed", "Undercut", "High and Tight", "Thinning", "Thinning Front", "Thinning Back", "Flat Top", "Buzzcut")
+
+var/list/hairsemisafe = list("Half-Shaved", "Half-Shaved Emo", "Long Side Emo", "Sideswept Hair", "Shaved Mohawk", "Tight Shaved Mohawk", "Naomi Mohawk", "Left Sidecut", "Right Sidecut", "Gentle 2", "Gentle 2 (Long)", "Gentle 2, Alternative", "Gentle 2, Alternative (Long)", "Neat", "Neat (Long)", "Donut Bun", "Shoulder Bob", "Messy", "Mohawk", "Drills, Side", "Low Bun", "Unkept", "Modern", "Bun", "Casual Bun", "Double-Bun", "Bangs", "Sleeze", "Medium Fade", "High Fade", "Tight Bun", "Coffee House Cut", "Bowl2", "Manbun", "Manbun", "Shaved Bun", "Librarian Bun", "Fade", "Krewcut", "Choppy (Short)", "Floof (Short)", "Sideshave", "Waxed", "Jenjen", "Fade (Grown)", "Swept", "Spiked")
+
+var/list/hairsortasafe = list("Shoulder-length Hair", "Long Hair", "Long Hair Alt", "Very Long Hair", "Long Fringe", "Longer Fringe", "Ponytail 1", "Ponytail 2", "Ponytail 3", "Ponytail 4", "Ponytail 5", "Ponytail 6", "Ponytail 7", "Side Ponytail", "Side Ponytail 2", "One Shoulder", "Tress Shoulder", "Pompadour", "Beehive", "Beehive 2", "Curls", "Long Emo", "Emo Fringe", "Flow Hair", "Overeye Long", "Feather", "Hitop","Adam Jensen Hair", "Gelled Back", "Gentle", "Spiky", "Kusanagi Hair", "Pigtails", "Odango", "Ombre", "Updo", "Drillruru", "Nitori", "Joestar", "Volaju", "80's", "Emo", "Devil Lock", "Reverse Mohawk", "Bowl", "Chin Length Bob", "Bob", "Bobcurl", "Bedhead", "Bedhead 2", "Quiff", "Parted", "Fringetail", "Dandy Pompadour", "Poofy", "Wavy Shoulder (Down)", "Wavy Shoulder (Ponytail)", "Rows", "Rows 2", "Grande Braid", "Medium Braid" ,)
+
+
+var/list/hairunsafe = list("Dreadlocks", "Afro", "Afro 2", "Big Afro", "Topknot", "Ronin", "Floorlength Braid", "Long Braid", "Long Braid 2", "Chrono", "Vegeta", "Nia", "Pomp III", "Superbowl", "Cactus")
+
+
+var/list/skrelltentacles = list("Skrell Average Tentacles", "Skrell Very Short Tentacles", "Skrell Very Long Tentacles")
```

Comment:
would these not be more tuggable by length

Reply:
No.